### ISSUE TITLE
ColMarTechCargoUniform is not mapped.

### ISSUE THREAD

**Issue by PursuitInAshes** [RMC-14/RMC-14#3085]
## Description
The `ColMarTechCargoUniform` is not mapped on the Almayer. This vendor provides surplus additional uniforms and extra radio keys for squads or channels like intel, req, jtac, so on.

**Reproduction**
Code search `ColMarTechCargoUniform`, if it was mapped it would show up.

**Additional context**
While I don't know if this was a intentional or an oversight, discussing [here](https://discord.com/channels/1168210010233376858/1261918674743197736/1261918674743197736). made me realize that it wasn't mapped on the Almayer and probably should be.

**Comment by Tunguso4ka** [comment]
```
searching for "ColMarTechCargoUniform" in ./

📄 founded in Resources/Maps/_RMC14/savannah.yml on lines: 25899
```

**Comment by Tunguso4ka** [comment]
>  Could you please check the Almayer?

I dont have access to Almayer.

**Comment by Tunguso4ka** [comment]
> Its on the savannah, but not almayer.

Almayer`s file is not public. I`l add this issue to a List of Almayer Problems.

**Comment by Tunguso4ka** [comment]
> It is. It can be found [here.](https://github.com/RMC-14/RMC-14/blob/master/Resources/Maps/_RMC14/almayer.yml) 

Thats Savannah under a different filename.
Both Almayer and LV624 are not public yet.

**Comment by OctoRocket** [comment]
> Both Almayer and LV624 are not public yet.
> 
> 

Why are the maps not public? Doesn't that conflict with the MIT license?

**Comment by Tunguso4ka** [comment]
> Why are the maps not public?

They are not done yet + hosts decision.

> Doesn't that conflict with the MIT license?

No, it doen't. MIT does not prohibit having "secrets" and private parts of code.
GPL3 do.

**Comment by OctoRocket** [comment]
> No, it doen't. MIT does not prohibit having "secrets" and private parts of code.
> 
> GPL3 do.

"Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:"

I do believe the maps are considered part of the software.

**Comment by Tunguso4ka** [comment]
> I do believe the maps are considered part of the software.

Anything that you can see on this github repo is licensed under MIT (unless stated otherwise), and anything you cant see is not.
Maps is not a part of software. Software works with and without this maps.

**Comment by OctoRocket** [comment]
If the server is running a modified version of this code, then it can't have the MIT license as the code the server is running is not public. If the server is running software with the MIT license, then the software it is running must have fully publicly accessible code. This seems to be not the case, as a file that is a regular part of the gameplay loop is missing from the repo. Furthermore, the repo is advertised as the software that the server is running. If the server has private files, then this is not the case.

**Comment by OctoRocket** [comment]
> Anything that you can see on this github repo is licensed under MIT (unless stated otherwise), and anything you cant see is not. Maps is not a part of software. Software works with and without this maps.

I believe that the software is defined as anything necessary for the game to be playable. Maps are essential to the game's playability and are therefore part of the software. The MIT license never defines what software is (other than including that documentation is part of the software) so I would have to look for legal cases regarding the license where software is more well defined.

**Comment by DrSmugleaf** [comment]
@OctoRocket 
The MIT license does not prohibit anything from being public.
Even if it did, it applies to code, not assets. Maps are an asset.
What you are thinking of is the AGPL, which we do not use.

**Comment by OctoRocket** [comment]
Are the assets not part of the software? They aren't licensed any other way, so wouldn't they also be licensed under MIT?

**Comment by QuietlyWhisper** [comment]
This codebase isn't AGPL

**Comment by DrSmugleaf** [comment]
> Are the assets not part of the software? They aren't licensed any other way, so wouldn't they also be licensed under MIT?

Again, let me repeat what I stated:
The MIT license does not prohibit anything from being public. What you are reading is the terms of what is made available. There is no implicit nor explicit promise on what is available under MIT.
Even if it did, it applies to code, not assets. Maps are an asset.
What you are thinking of is the AGPL, which we do not use.
If you want the private maps to be public, you are free to map them yourself like I did.

If you wish to discuss this further, discuss it on Discord, do not spam an unrelated issue with this.
